install/kubernetes/cilium/templates/cilium-operator/role.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.operator.enabled .Values.serviceAccounts.operator.create .Values.ingressController.enabled .Values.ingressController.envoySecretNamespace }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cilium-opeartor-secrets
+  namespace: {{ .Values.ingressController.envoySecretNamespace }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - create
+  - delete
```

Comment:
probably needs `update` as well

Reply:
you are right, I missed update in this file, local testing was done with update permission :)